script/__snapshots__/migrate-test-e2e.js.snap

```diff
@@ -0,0 +1,847 @@
+ 
+ See [\`.github/CONTRIBUTING.md\`](./.github/CONTRIBUTING.md), then [\`.github/DEVELOPMENT.md\`](./.github/DEVELOPMENT.md).
+@@ -128,3 +91,7 @@ Thanks! 💖
+ 
+ <!-- ALL-CONTRIBUTORS-LIST:END -->
+ <!-- spellchecker: enable -->
++
++<!-- You can remove this notice if you don't want it 🙂 no worries! -->
++
++> 💙 This package was templated with [\`create-typescript-app\`](https://github.com/JoshuaKGoldberg/create-typescript-app)."
+`;
+
+exports[`verify cspell.json 1`] = `
+"--- a/cspell.json
++++ b/cspell.json
+@@ -1,40 +1,27 @@
+ {
+ 	"dictionaries": ["typescript"],
+ 	"ignorePaths": [
+-		"./coverage*",
+ 		".github",
+ 		"CHANGELOG.md",
++		"coverage",
+ 		"lib",
+ 		"node_modules",
+-		"pnpm-lock.yaml",
+-		"./script/__snapshots__"
++		"pnpm-lock.yaml"
+ 	],
+ 	"words": [
+-		"allcontributors",
+-		"apexskier",
+-		"arethetypeswrong",
+ 		"Codecov",
+ 		"codespace",
+ 		"commitlint",
+ 		"contributorsrc",
+ 		"conventionalcommits",
+-		"execa",
+-		"infile",
+-		"joshuakgoldberg",
+ 		"knip",
+ 		"lcov",
+ 		"markdownlintignore",
+-		"mtfoley",
+-		"npmignore",
+-		"npmjs",
+ 		"npmpackagejsonlintrc",
+ 		"outro",
+ 		"packagejson",
+ 		"quickstart",
+-		"tada",
+ 		"tsup",
+-		"Unstaged",
+-		"vitest",
+ 		"wontfix"
+ 	]
+ }"
+`;
+
+exports[`verify knip.jsonc 1`] = `
+"--- a/knip.jsonc
++++ b/knip.jsonc
+@@ -1,17 +1,6 @@
+ {
+ 	"$schema": "https://unpkg.com/knip@latest/schema.json",
+-	"entry": [
+-		"src/index.ts!",
+-		"src/guide/index.ts",
+-		"src/initialize/index.ts",
+-		"src/migrate/index.ts",
+-		"script/*e2e.js"
+-	],
+-	"ignoreBinaries": ["gh"],
+-	"ignoreDependencies": ["script"],
+-	"ignoreExportsUsedInFile": {
+-		"interface": true,
+-		"type": true
+-	},
+-	"project": ["src/**/*.ts!", "script/**/*.js"]
++	"entry": ["src/index.ts!"],
++	"ignoreExportsUsedInFile": { "interface": true, "type": true },
++	"project": ["src/**/*.ts!"]
+ }"
+`;
+
+exports[`verify package.json 1`] = `
+"--- a/package.json
++++ b/package.json
+@@ -13,9 +13,7 @@
+ 	},
+ 	"type": "module",
+ 	"main": "./lib/index.js",
+-	"bin": "./bin/index.js",
+ 	"files": [
+-		"bin/",
+ 		"lib/",
+ 		"package.json",
+ 		"LICENSE.md",
+@@ -24,7 +22,6 @@
+ 	"scripts": {
+ 		"build": "tsup",
+ 		"format": "prettier \\"**/*\\" --ignore-unknown",
+-		"initialize": "tsx ./src/bin/index.js --mode initialize",
+ 		"lint": "eslint . .*js --max-warnings 0",
+ 		"lint:knip": "knip",
+ 		"lint:md": "markdownlint \\"**/*.md\\" \\".github/**/*.md\\" --rules sentences-per-line",
+@@ -34,9 +31,6 @@
+ 		"prepare": "husky install",
+ 		"should-semantic-release": "should-semantic-release --verbose",
+ 		"test": "vitest",
+-		"test:create": "node script/create-test-e2e.js",
+-		"test:initialize": "node script/initialize-test-e2e.js",
+-		"test:migrate": "vitest run -r script/",
+ 		"tsc": "tsc"
+ 	},
+ 	"lint-staged": {
+@@ -107,7 +101,7 @@
+ 		"vitest": "^1.0.2",
+ 		"yaml-eslint-parser": "^1.2.2"
+ 	},
+-	"packageManager": "pnpm@8.12.1",
++	"packageManager": "pnpm@8.7.0",
```

Comment:
Heh, and in CI:

```plaintext
2023-12-30T06:48:46.9914884Z    	},
2023-12-30T06:48:46.9915527Z - -	"packageManager": "pnpm@8.12.1",
2023-12-30T06:48:46.9916260Z + -	"packageManager": "pnpm@8.13.1",
2023-12-30T06:48:46.9916878Z   +	"packageManager": "pnpm@8.7.0",
```

This'll probably start failing as soon as a new pnpm version is out. Lovely. That can be a followup.